### PR TITLE
Refactors the CargoOrderDataManager into the CargoConsoleSystem

### LINKED ISSUE #2798: Cargo requests do not clear after the round ends

### PR DESCRIPTION
**Pull request by Radrark**
Then I made the CargoConsoleSystem resettable to clear the cargo orders after the round ends.

Fixes #2798

### CODE REVIEW

File: Content.Server/GameObjects/EntitySystems/CargoConsoleSystem.cs [hunk 1]
```diff
@@ -36,9 +41,12 @@ public class CargoConsoleSystem : EntitySystem
         /// </summary>
         public CargoBankAccount StationAccount => GetBankAccount(0);
 
+        public CargoOrderDatabase StationOrderDatabase => GetOrderDatabase(0);
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/EntitySystems/CargoConsoleSystem.cs:44]: @ShadowCommander would there be more than 1 order db in a round?

**Review by metalgearsloth — APPROVED** [review]

**Comment by metalgearsloth** [comment]
Would you also mind fixing the cargo console component's OnRemove event as well?

**Comment by Radrark** [comment]
> Would you also mind fixing the cargo console component's OnRemove event as well?

Done

**Comment by metalgearsloth** [comment]
Legend, just waiting on SC to comment

File: Content.Server/GameObjects/EntitySystems/CargoConsoleSystem.cs (hunk 1 above)
Comment (ShadowCommander) [reply Content.Server/GameObjects/EntitySystems/CargoConsoleSystem.cs:44]: When I designed it, I had multiple stations in mind. It's not necessary, but it'd be cool to be able to have two stations with separate cargo databases.

**Review by ShadowCommander — COMMENTED** [review]
I'll take a closer look when I get home in two hours.

**Comment by metalgearsloth** [comment]
@ShadowCommander no news is good news right

**Pull request merged**